src/seedu/addressbook/AddressBook.java

```diff
@@ -1154,6 +1172,19 @@ private static String removePrefixSign(String s, String sign) {
         return s.replace(sign, "");
     }
 
+    /* peixuan's code (T2A2 Abstraction) */
+    /**
+    * Removes prefix from the given fullString if prefix occurs at the start of the string.
+    */
+    /*
+    private static String removePrefix(String fullString, String prefix) { 
+        if (fullString.startsWith(prefix)) {
+            return fullString.replaceFirst(fullString, prefix); 
+        }
+        return fullString;
+    }
+    */
+
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.